src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/ClientBaseTests.cs

```diff
@@ -125,31 +125,92 @@ public static void ClientMessageInspector_Verify_Invoke()
 
     [Fact]
     [OuterLoop]
-    public static void ClientBaseOfT_Call()
+    public static void ClientBaseOfT_Sync_RoundTrip_Check_CommunicationState()
```

Comment:
Okay, I give up about checking communication state :smile: .  But then why doesn't every other e2e test do the same?  I think it is good practice to make each test verify exactly one thing.  @StephenBonikowsky and I had a conversation and agreed scenario tests should test only one thing.  Pre-conditions related to setting up for the test and post-conditions tangential to the main test should be assumed to be correct and not explicitly tested. They belong in dedicated tests of their own.